worldedit-sponge/build.gradle

```diff
@@ -0,0 +1,90 @@
+apply plugin: 'eclipse'
+apply plugin: 'idea'
+
+buildscript {
+    repositories {
+        mavenCentral()
+        maven { url = "http://files.minecraftforge.net/maven" }
+        maven { url = "https://oss.sonatype.org/content/repositories/snapshots/" }
+        jcenter()
+    }
+
+    dependencies {
+        classpath 'net.minecraftforge.gradle:ForgeGradle:2.1-SNAPSHOT'
+    }
+}
+
+apply plugin: 'net.minecraftforge.gradle.forge'
```

Comment:
Er, if we use the spongevanilla workspace as minecrell says or whatever, does that also mean it's compatible with spongeforge then? i was under the impression that being compatible with both would mean only using the api (or maybe things in common at most - but that requires compiling two different ways?)